src/Common/src/Internal/Runtime/GCDescEncoder.cs

```diff
@@ -1,122 +1,117 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think we lost a BOM. This was not actually a rename, but I guess GIT was smart enough to figure out where the code moved from.